.readthedocs.yml

```diff
@@ -5,26 +5,15 @@
 # Required
 version: 2
 
-# Set the version of Python and other tools you might need
 build:
-  os: ubuntu-22.04
+  os: ubuntu-24.04
   tools:
-    python: "3.11"
-
-  # Unshallow the git clone to allow vcs versioning to pick
-  # up tags properly
-  jobs:
-    post_checkout:
-      - git fetch --unshallow
-
-# Build documentation in the docs/ directory with Sphinx
-sphinx:
-  configuration: docs/conf.py
-
-python:
-  install:
-    - method: pip
-      path: .
-      extra_requirements:
-        - docs
-        - pyproject
+    python: "3.13"
+  commands:
```

Comment:
Could put this in `build.jobs.pre_build` instead and minimize the diff. Or do you need `uv run sphinx`?